pkg/configuration/configuration.go

```diff
@@ -222,11 +223,27 @@ func (ev *extendedViper) Set(key string, value interface{}) {
 	}
 }
 
+func bindEnvSnykErr(key string, err error) snyk_errors.Error {
+	return snyk_errors.Error{
+		Title:          "Unable to bind env",
+		Classification: "ACTIONABLE",
+		Level:          "warning",
+		Detail:         fmt.Sprintf("Unable to bind env: %v. Reason: %v", key, err),
+	}
+}
+
 func (ev *extendedViper) get(key string) interface{} {
 	ev.mutex.Lock()
 	defer ev.mutex.Unlock()
 
-	// try to lookup given key
+	// ensure we only get Snyk related keys (prefixed with SNYK_)
+	if ev.GetKeyType(key) == EnvVarKeyType {
+		err := ev.viper.BindEnv(key, strings.ToUpper(key))
+		if err != nil {
+			return bindEnvSnykErr(key, err)
+		}
+	}
+	// try to lookup given key with SNYK_ prefix
```

Comment:
Suggestion: instead of commenting on an implementation limitation, why don't we make supported prefixes configurable. That shouldn't be too much work.

Reply:
Alternative take, we support only the `SNYK_` prefix and use this as an opportunity to implement consistency by requiring all env vars we want to support to be prefixed with `SNYK_`?